stl/inc/xstring

```diff
@@ -4511,16 +4554,19 @@ _NODISCARD bool operator==(
     return _Left._Equal(_Right);
 }
 
+#if !_HAS_CXX20
 template <class _Elem, class _Traits, class _Alloc>
 _NODISCARD bool operator==(_In_z_ const _Elem* const _Left, const basic_string<_Elem, _Traits, _Alloc>& _Right) {
     return _Right._Equal(_Left);
 }
+#endif // !_HAS_CXX20

```

Comment:
Could we move this past the next overload and merge it into the other `!_HAS_CXX20` conditional just to reduce the total preprocesor noise?